strider/profiler.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+import time
+import uuid
+
+from fastapi.staticfiles import StaticFiles
+from fastapi.responses import HTMLResponse
+import yappi
+
+from .server import APP
+
+class DownloadStaticFiles(StaticFiles):
+    """
+    Wrapper on StaticFiles to serve files that are automatically
+    downloaded using the Content-Disposition header
+    """
+    def file_response(*args, **kwargs):
+        resp = StaticFiles.file_response(*args, **kwargs)
+        resp.headers["Content-Disposition"] = "attachment"
+        return resp
+
+# Create directory to store request profile data
+PROFILE_DIRECTORY = "/profiles"
```

Comment:
Do we mean to have an absolute path here?

Reply:
Yes. This is in the docker container, so an absolute path in the root filesystem is fine.

Reply:
This _can_ be in a Docker container, but it can also be run locally if we choose a directory we have permission to create.

Reply:
Who runs things outside of Docker?? Should be fixed now...